app/src/main/java/org/sagebionetworks/bridge/researchstack/CrfTaskHelper.java

```diff
@@ -48,12 +49,20 @@
 
     public static final String ANSWERS_FILENAME = "answers";
 
-    static HashMap<String, String> CRF_RESULT_CONVERSION_MAP = new HashMap<String, String>() {{
-        put("HeartRateCamera_heartRate.before", "heartRate_before_recorder");
-        put("HeartRateCamera_heartRate.after",  "heartRate_after_recorder");
-        put("motion_stairStep",                 "stairStep_motion");
-        put("location_run",                     "location");
-    }};
+    static final Map<String, String> CRF_FILENAME_CONVERSION_MAP =
+            ImmutableMap.<String, String>builder()
+                    .put("HeartRateCamera_heartRate.before",    "heartRate_before_recorder")
```

Comment:
This map is great for matching iOS.  Maybe one day they will have a map to match us!